docs/docusaurus/docs/core/installation_and_setup/components_how_to_configure_a_validation_result_store_in_amazon_s3/_identify_your_data_context_validation_results_store.mdx

```diff
@@ -0,0 +1,9 @@
+import TechnicalTag from '../../../../term_tags/_tag.mdx';
+
+Your <TechnicalTag tag="validation_result_store" text="Validation Results Store" /> configuration is in your <TechnicalTag tag="data_context" text="Data Context" />.
```

Comment:
You can go through anything where you find <TechnicalTag and update it like this:
```suggestion
Your Validation Results Store configuration is in your Data Context".
```
Basically: Delete everything except for the value of `text=` in the TechnicalTags.  That will let you also remove the `import TechnicalTag from...` statements, and should clear up a lot of the broken links.

Reply:
@Rachel-Reverie - done.